package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.1",
```

Comment:
If this PR merges first, I'd take 6.7.0, then let the other PR author know they need to update to 6.7.1 before they merge.